app/models/ruby_object.rb

```diff
@@ -56,6 +56,20 @@ def ruby_methods
     @ruby_methods ||= body[:methods].collect { |m| RubyMethod.new(m) }
   end
 
+  def superclass
+    return @superclass if defined?(@superclass)
+
+    value = body[__method__]
```

Comment:
Is there a reason you used `__method__` here (and in `included_modules`)? This is the first time I've ever seen this. 

Based on the docs I just read, it seems like you could have just used a symbol literal of `:superclass`. Is `__method__` better for some reason?

Reply:
> Is there a reason you used `__method__` here (and in `included_modules`)?

DRY.

> Based on the docs I just read, it seems like you could have just used a symbol literal of `:superclass`. Is `__method__` better for some reason?

Yes, I could. Yes, it's better for DRY reason.